server/server.go

```diff
@@ -49,6 +50,11 @@ func Run(conf *config.Config) error {
 	case <-signalCh:
 		log.Info("signal detected")
 	}
+
+	if err = subscriber.Close(); err != nil {
+		return err
+	}
+
```

Comment:
If subscriber is needed only for close, you can close the subscriber in `cmd/start.go` using defer function.